server/datastore/mysql/hosts.go

```diff
@@ -1926,68 +1943,83 @@ WHERE
 // Aggregated stats for os versions are stored by team id with 0 representing the global case
 // If existing team has no hosts, we explicity set the json value as an empty array
 func (ds *Datastore) UpdateOSVersions(ctx context.Context) error {
+	// TODO: counts will be lagging until offline hosts report back with detail query
```

Comment:
What's there TODO here?

Reply:
It's a reminder for me coordinate with product on how to message this to the end user. I've discussed this with Luke and will be following up with Noah so I'll remove the TODO now.